src/lightning_app/utilities/app_helpers.py

```diff
@@ -402,3 +402,66 @@ def default(self, obj: Any) -> Any:
         if callable(getattr(obj, "__json__", None)):
             return obj.__json__()
         return json.JSONEncoder.default(self, obj)
+
+
+def _state_dict(flow: "LightningFlow"):
```

Comment:
I don't understand the use of these 2 helpers.

Which component calls these?
Why is this logic not part of `flow.state_dict`?